packages/svelte/src/internal/client/proxy/proxy.js

```diff
@@ -32,13 +33,13 @@ const is_frozen = Object.isFrozen;
  * @param {T} value
  * @returns {T}
  */
-export function proxy(value) {
+export function proxy(value, immutable = true) {
```

Comment:
The JSDoc is missing a `@param {boolean} [immutable]` - also: what is this needed for exactly? I don't quite understand.